configure.py

```diff
@@ -1416,7 +1416,7 @@ def DolphinLib(lib_name, objects):
     ActorRel(NonMatching, "d_a_bwds"),
     ActorRel(NonMatching, "d_a_daiocta"),
     ActorRel(NonMatching, "d_a_daiocta_eye"),
-    ActorRel(NonMatching, "d_a_deku_item"),
+    ActorRel(NonMatching, "d_a_deku_item", extra_cflags=['-pragma "nosyminline on"']),
```

Comment:
```suggestion
    ActorRel(Matching,    "d_a_deku_item", extra_cflags=['-sym off']),
```
not sure if you already tried it, but does this allow it to match?

if it still doesn't match for some reason, you could do a build (while this is incorrectly marked as matching) and then try a binary comparison of `tww/build/GZLE01/d_a_deku_item/d_a_deku_item.rel` with the original REL from the vanilla game